jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/pagination.feature

```diff
@@ -0,0 +1,280 @@
+      | group              | group1              |
+      | apacheHttpdMediaId | apache-httpd-2.4.20 |
+      | statusPath         | /apache_pb.png      |
+    And I created a web server with the following parameters:
+      | webserverName      | 27                  |
+      | hostName           | localhost           |
+      | portNumber         | 80                  |
+      | httpsPort          | 443                 |
+      | group              | group1              |
+      | apacheHttpdMediaId | apache-httpd-2.4.20 |
+      | statusPath         | /apache_pb.png      |
+    And I created a web server with the following parameters:
+      | webserverName      | 28                  |
+      | hostName           | localhost           |
+      | portNumber         | 80                  |
+      | httpsPort          | 443                 |
+      | group              | group1              |
+      | apacheHttpdMediaId | apache-httpd-2.4.20 |
+      | statusPath         | /apache_pb.png      |
+    And I created a web server with the following parameters:
+      | webserverName      | 29                  |
+      | hostName           | localhost           |
+      | portNumber         | 80                  |
+      | httpsPort          | 443                 |
+      | group              | group1              |
+      | apacheHttpdMediaId | apache-httpd-2.4.20 |
+      | statusPath         | /apache_pb.png      |
+    And I click on the sort button with attribute "Name"
+    And I see "29" in the webserver table
+
+
+  Scenario: Pagination within groups
+
+
+    Given I logged in
+    And I am in the configuration tab
+    And I am in the group tab
+    And I created a group with the name "A1"
+    And I created a group with the name "A2"
+    And I created a group with the name "A3"
+    And I created a group with the name "A4"
+    And I created a group with the name "A5"
+    And I created a group with the name "A6"
+    And I created a group with the name "A7"
+    And I created a group with the name "A8"
+    And I created a group with the name "A9"
+    And I created a group with the name "A10"
+    And I created a group with the name "A11"
+    And I created a group with the name "A12"
+    And I created a group with the name "A13"
+    And I created a group with the name "A14"
+    And I created a group with the name "A15"
+    And I created a group with the name "A16"
+    And I created a group with the name "A17"
+    And I created a group with the name "A18"
+    And I created a group with the name "A19"
+    And I created a group with the name "A20"
+    And I created a group with the name "A21"
+    And I created a group with the name "A22"
+    And I created a group with the name "A23"
+    And I created a group with the name "A24"
+    And I created a group with the name "A25"
+    And I created a group with the name "A26"
+    And I click the next page button
+    Then I see the text "Page 2/2"
```

Comment:
No When ? Instead of having 2 "Then" we should just concatenate with And e.g. Then And And And ...